CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 ## 1.11.0
 
+Enhancements:
+
+- Adds `WithAllowsPooling` option
+- Adds `WithURL` option
+
 Changes:
 
-- Adds `WithTrafficPolicy` function, replacing `WithPolicyString`
+- Replaces `WithPolicyString` option with `WithTrafficPolicy`
```

Comment:
Can we change the wording of this to indicate that `WithPolicyString` is just _deprecated_ and not replaced entirely? Was about to come in hot with the "no breaking changes without a major version" hammer, but then discovered that that isn't actually the case :sweat_smile:

Reply:
yeah you're right, this wording isn't so good. I'll probably move `WithTrafficPolicy` to enhancements and leave a deprecation message for `WithPolicyString`